debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/ClassSummaryHelper.java

```diff
@@ -67,26 +69,26 @@ public void printStatistics(PrintStream out) {
 				numberOfClasses, totalSize, totalSize / 1024));
 
 		// Print the sections
-		out.println(String.format("%-40s%-8s %-8s %-6s", "Section", "Byte",
-				"kB", "%"));
+		out.println(String.format("%-39s %-8s %-8s %-6s",
+				"Section", "Byte", "kB", "%"));
 		for (Map.Entry<String, Long> entry : data.entrySet()) {
 			out.println(printSize(entry.getKey(), entry.getValue(), totalSize));
 		}
 	}
 
-	private void appendSizeStat(Map<String, Long> sizeStat,
+	private static void appendSizeStat(Map<String, Long> sizeStat,
 			J9ClassRegionNode regionNode, String parentName) {
 		String section = parentName;
 		final J9ClassRegion nodeValue = regionNode.getNodeValue();
 		if (nodeValue != null) {
 			if (nodeValue.getType() == SlotType.J9_SECTION_START
 					|| nodeValue.getType() == SlotType.J9_Padding) {
 				section = section + "/" + nodeValue.getName();
-				if (sizeStat.containsKey(section)) {
-					sizeStat.put(section, new Long(sizeStat.get(section)
-							+ nodeValue.getLength()));
+				Long length = sizeStat.get(section);
+				if (length != null) {
+					sizeStat.put(section, Long.valueOf(length.longValue() + nodeValue.getLength()));
 				} else {
-					sizeStat.put(section, new Long(nodeValue.getLength()));
+					sizeStat.put(section, Long.valueOf(nodeValue.getLength()));
```

Comment:
Seems unnecessary to create a new Long instead of using the existing one.

Reply:
Long objects are immutable: this is really the only way to do this.